lib/Jsrt/ChakraCommon.h

```diff
@@ -71,6 +71,10 @@ typedef BYTE* ChakraBytePtr;
 #define CHAKRA_API extern "C" SET_API_VISIBILITY JsErrorCode
 #else
 #define CHAKRA_API extern     SET_API_VISIBILITY JsErrorCode
+#include <stdbool.h>
+#endif
+#if !defined(__cplusplus) || (__cplusplus <= 199711L)
```

Comment:
nit pick: Could you please insert a blank line before this `#if` for readability?